### ISSUE TITLE
'zope-interface>=5' distribution was not found

### ISSUE THREAD

**Issue by xuoguoto** [buildbot/buildbot#7073]
Trying to install buildbot on a fresh Ubuntu 22.04 server. After installing the recommended debs in tutorial, I installed buildbot (in a venv) using `pip install 'buildbot[bundle]'` which installed the following packages:

`Successfully installed Jinja2-3.1.2 Mako-1.2.4 MarkupSafe-2.1.3 PyJWT-2.8.0 Twisted-23.8.0 alembic-1.12.0 attrs-23.1.0 autobahn-23.6.2 automat-22.10.0 buildbot-3.9.0 buildbot-console-view-3.9.0 buildbot-grid-view-3.9.0 buildbot-waterfall-view-3.9.0 buildbot-worker-3.9.0 buildbot-www-3.9.0 cffi-1.15.1 constantly-15.1.0 cryptography-41.0.3 future-0.18.3 greenlet-2.0.2 hyperlink-21.0.0 idna-3.4 incremental-22.10.0 msgpack-1.0.5 pycparser-2.21 python-dateutil-2.8.2 pyyaml-6.0.1 six-1.16.0 sqlalchemy-1.4.49 txaio-23.1.1 typing-extensions-4.7.1 zope.interface-6.0`

I can see that `zope.interface-6.0` is installed and buidbot version is 3.9.0

When trying to create master I get the following error:

```python
(sandbox) user@dell02kvmi03:~/buildbot$ buildbot create-master master
mkdir /home/user/buildbot/master
creating /home/user/buildbot/master/master.cfg.sample
creating database (sqlite:///state.sqlite)
buildmaster configured in /home/user/buildbot/master
Unhandled error in Deferred:

Traceback (most recent call last):
  File "/home/user/buildbot/sandbox/lib/python3.10/site-packages/buildbot/scripts/create_master.py", line 84, in createDB
    master = BuildMaster(config['basedir'])
  File "/home/user/buildbot/sandbox/lib/python3.10/site-packages/buildbot/master.py", line 102, in __init__
    self._services_d = self.create_child_services()
  File "/home/user/buildbot/sandbox/lib/python3.10/site-packages/twisted/internet/defer.py", line 2245, in unwindGenerator
    return _cancellableInlineCallbacks(gen)
  File "/home/user/buildbot/sandbox/lib/python3.10/site-packages/twisted/internet/defer.py", line 2157, in _cancellableInlineCallbacks
    _inlineCallbacks(None, gen, status, _copy_context())
--- <exception caught here> ---
  File "/home/user/buildbot/sandbox/lib/python3.10/site-packages/twisted/internet/defer.py", line 1997, in _inlineCallbacks
    result = context.run(gen.send, result)
  File "/home/user/buildbot/sandbox/lib/python3.10/site-packages/buildbot/master.py", line 188, in create_child_services
    self.www = wwwservice.WWWService()
  File "/home/user/buildbot/sandbox/lib/python3.10/site-packages/buildbot/www/service.py", line 196, in __init__
    self.apps = get_plugins('www', None, load_now=True)
  File "/home/user/buildbot/sandbox/lib/python3.10/site-packages/buildbot/plugins/db.py", line 356, in get_plugins
    return _DB.add_namespace(namespace, interface, check_extras, load_now)
  File "/home/user/buildbot/sandbox/lib/python3.10/site-packages/buildbot/plugins/db.py", line 306, in add_namespace
    tempo.load()
  File "/home/user/buildbot/sandbox/lib/python3.10/site-packages/buildbot/plugins/db.py", line 242, in load
    self._tree.load()
  File "/home/user/buildbot/sandbox/lib/python3.10/site-packages/buildbot/plugins/db.py", line 112, in load
    child.load()
  File "/home/user/buildbot/sandbox/lib/python3.10/site-packages/buildbot/plugins/db.py", line 45, in load
    self._value = self._loader(self._entry)
  File "/home/user/buildbot/sandbox/lib/python3.10/site-packages/buildbot/plugins/db.py", line 214, in _load_entry
    raise PluginDBError('Requirements are not satisfied '
buildbot.errors.PluginDBError: Requirements are not satisfied for buildbot.www:base: The 'zope-interface>=5' distribution was not found and is required by Twisted
```

As far as I can see, packages are all present, what could be wrong here?

**Comment by p12tic** [comment]
This is a known bug in Twisted or one of its requirements: https://github.com/twisted/twisted/issues/11935

**Comment by p12tic** [comment]
Note that I can't reproduce this bug on Debian, so it's something specific to Ubuntu.

**Comment by p12tic** [comment]
I think this is fixed (well, really only worked around) in https://github.com/buildbot/buildbot/pull/7078.

**Comment by p12tic** [comment]
Proper fix is in https://github.com/buildbot/buildbot/pull/7035, but it needs more work.

**Comment by p12tic** [comment]
Workaround released in Buildbot 3.9.2.

**Comment by xuoguoto** [comment]
After upgrading it's working fine:

`Successfully installed Twisted-22.10.0 buildbot-3.9.2 buildbot-console-view-3.9.2 buildbot-grid-view-3.9.2 buildbot-waterfall-view-3.9.2 buildbot-worker-3.9.2 buildbot-www-3.9.2`

``` bash
$ buildbot create-master master
updating existing installation
buildbot.tac already exists and is correct
creating /home/user/buildbot/master/master.cfg.sample
creating database (sqlite:///state.sqlite)
buildmaster configured in /home/user/buildbot/master
```

**Comment by yan12125** [comment]
> Proper fix is in #7035, but it needs more work.

That one is merged. Maybe the work-around can be dropped on master branch?
